toc/rfc/rfc-draft-cc-blobstore-storage-cli.md

```diff
@@ -0,0 +1,68 @@
+# Meta
+[meta]: #meta
+- Name: Cloud Controller Blobstore Type: storage-cli
+- Start Date: 2025-07-18
+- Author(s): @johha, @stephanme
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: [community #1253](https://github.com/cloudfoundry/community/pull/1253)
+
+
+## Summary
+
+Add a blobstore type `storage-cli` to the Cloud controller that is based on the Bosh storage CLIs. Long-term, the `storage-cli` blobstore type shall replace the blobstore type `fog`.
```

Comment:
Should we mention here that a new storage-cli will be created which will replace all bosh storage CLIs in the long run?

Reply:
done